conf.py

```diff
@@ -106,7 +106,8 @@ def reset_seeds(gallery_conf, fname):
     'first_notebook_cell': ("# For tips on running notebooks in Google Colab, see\n"
                             "# https://pytorch.org/tutorials/beginner/colab\n"
                             "%matplotlib inline"),
-    'reset_modules': (reset_seeds)
+    'reset_modules': (reset_seeds),
+    'ignore_pattern': r'_torch_export_nightly_tutorial.py'
```

Comment:
Do you need this if it's already put in the NOT_RUN list?

Reply:
The not run list prevents the dummy tutorial from being executed and failing tests. However, it is showing up in the tutorial search tool.